charts/sonarqube/README.md

```diff
@@ -55,6 +55,7 @@ The following table lists the configurable parameters of the _SonarQube_ chart a
 | `extraVolumeMounts`                | Additional volume mounts for the _sonarqube_ container.                                                                         | `[]`                                |
 | `resources`                        | Resource requests and limits for the _sonarqube_ container.                                                                     | `{}`                                |
 | `extraInitContainers`              | Additional init containers for the pod, these can be templated.                                                                 | `[]`                                |
+| `extraContainers`              | Additional containers for the pod, these can be templated.                                                                 | `[]`                                |
```

Comment:
Could you remove the reference to templating as it's not enabled here (and I'm removing it from the init containers as it doesn't work as expected).